Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -3485,6 +3497,22 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   }
 }
 
+extension Syntax {
+  var isPostfixIfConfig: Bool {
+    var this: Syntax? = self
+
+    while this?.parent != nil {
```

Comment:
Do we need to check all the ancestors, not just the immediate parent? Is this required to make nested clauses have the right indentation? (If it is, that's fine, I just wanted to verify the reason.)